docs/spfx/supported-extensibility-platforms-overview.md

```diff
@@ -20,6 +20,8 @@ Learn more about using SPFx on SharePoint Online: [SPFx support for SharePoint O
 > [!IMPORTANT]
 > SharePoint Server Subscription Edition (SE), announced as a preview in July 2021, supports the same SPFx versions as SharePoint Server 2019.
```

Comment:
Shouldn't this get removed as SPSE 23H1 added support for newer versions than what was supported in SP 2019?